ecommerce/src/main/java/com/algolia/instantsearch/examples/ecommerce/EcommerceActivity.java

```diff
@@ -75,11 +75,9 @@ protected void onStop() {
     @Override
     protected void onDestroy() {
         filterResultsWindow.dismiss();
+        searcher.destroy();
         toggleArrow(buttonFilter, false);
         super.onDestroy();
-        RefWatcher refWatcher = EcommerceApplication.getRefWatcher(this);
-        refWatcher.watch(this);
-        refWatcher.watch(findViewById(R.id.hits));
```

Comment:
Why removing this? I thought we would leave LeakCanary enabled (until we can automate leak detection, it's better than nothing).

Reply:
Ah... oups ?!

Reply:
Done